rotkehlchen/api/rest.py

```diff
@@ -4474,6 +4479,22 @@ def export_history_events(
 
         return api_response(OK_RESULT, status_code=HTTPStatus.OK)
 
+    def _invalidate_accounting_rule_cache(
```

Comment:
you don't invalidate the entire cache. I got confused by the name
```suggestion
    def _invalidate_cache_for_accounting_rule(
```